src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -342,7 +342,7 @@
                 symbol.RefKind,
                 isExtension: symbol is { Ordinal: 0, ContainingSymbol: IMethodSymbol { IsExtensionMethod: true } },
                 symbol.IsParams,
-                symbol.ScopedKind == ScopedKind.ScopedRef);
+                isScoped: symbol is { RefKind: RefKind.Ref, ScopedKind: ScopedKind.ScopedRef});
```

Comment:
>RefKind.Ref [](http://example.com/codeflow?start=47&length=11)

This should probably include `in` as well: `RefKind.Ref or RefKind.In`

Reply:
Is it important to handle `ref struct` cases as well? If so, perhaps:

```
isScoped: symbol is
    { RefKind: RefKind.Ref or RefKind.In, ScopedKind: ScopedKind.ScopedRef } or
    { Type.IsRefLineType: true, ScopedKind: ScopedKind.Value }
```

Reply:
This is just for parameters and determines if the parameter gets the `scoped` keyword.  Would these suggestions apply to parameters?  If you can suggest all the different syntax to add to tests to cover these cases that might be a good way to fix this.

Reply:
Got it, I see how these are needed given the tests you suggested 👍